app/assets/javascripts/backbone/views/application_view.coffee

```diff
@@ -8,10 +8,12 @@ class MS.Views.ApplicationView extends Backbone.View
   initialize: ->
     MS.loadingIndicator = @$('#glb-load-ind')
     MS.errorModal = @$('#glb-error-modal')
+    MS.errorModalDefaultText = I18n.t('error_notification')
```

Comment:
Does this need to be a global? The above two do because they are accessed from other classes. But I don't think that's the case here?